src/Intersection.ts

```diff
@@ -89,21 +99,25 @@ export class Intersection {
    * @returns
    */
   static isPointInPolygon(point: Point, points: Point[]) {
-    const other = new Point(point).setX(
-      Math.min(point.x - 1, ...points.map((p) => p.x))
-    );
```

Comment:
you said this is a major perf hit so it is removed in favor of solid math

Reply:
Since you pointed out that you need this in production, i m asking, what is this doing more than fabric already does? my question is very simple If it was just a perf bump you wouldn't need it.

i have no issue merging it, i just need to try to get out some extra bytes, but my question is really, how a performance improvement become needed? it means that is more than performances no?

Reply:
Yes
Connecting lines
It is needed for it. I need to get an intersectiin for the side of the control I am connecting to. That is a ray intersection.
Needed especially for elbow (90deg) connectors